apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -480,6 +547,17 @@ defmodule OMG.Watcher.ExitProcessor.Core do
       get_ifes_with_competitors(request, state)
       |> Enum.map(fn txbytes -> %Event.NonCanonicalIFE{txbytes: txbytes} end)
 
+    invalid_piggybacks =
+      get_invalid_piggybacks(request, state)
+      |> Enum.map(fn {txbytes, inputs, outputs} ->
+        %Event.InvalidPiggyback{txbytes: txbytes, inputs: inputs, outputs: outputs}
+      end)
+
+    # TODO: late piggybacks are critical
+    late_invalid_piggybacks = []
+
+    has_no_late_invalid_exits = has_no_late_invalid_exits && Enum.empty?(late_invalid_piggybacks)
```

Comment:
maybe safer to put both `Enum.empties?` on the same line here instead of rebinding?

Reply:
done